hack/tools/tilt-prepare/main.go

```diff
@@ -70,17 +70,23 @@ var (
 	rootPath      string
 	tiltBuildPath string
 
-	toolsFlag            = pflag.StringSlice("tools", []string{}, "list of tools to be created; each value should correspond to a make target")
-	certManagerFlag      = pflag.Bool("cert-manager", false, "prepare cert-manager")
-	kustomizeBuildsFlag  = pflag.StringSlice("kustomize-builds", []string{}, "list of kustomize build to be run; each value should be in the form name:path")
-	providersBuildsFlag  = pflag.StringSlice("providers", []string{}, "list of providers to be installed; each value should be in the form name:path")
-	allowK8SContextsFlag = pflag.StringSlice("allow-k8s-contexts", []string{}, "Specifies that Tilt is allowed to run against the specified k8s context name; Kind is automatically allowed")
 	tiltSettingsFileFlag = pflag.String("tilt-settings-file", "./tilt-settings.yaml", "Path to a tilt-settings.(json|yaml) file")
```

Comment:
Can you please adjust the example usage in l58-66 to reflect the new flags?

Reply:
done